spec/decorators/course_decorator_spec.rb

```diff
@@ -243,6 +243,12 @@
 
         it { is_expected.to eq("Student finance if you’re eligible") }
       end
+
+      context "Courses excluded from bursaries" do
+        let(:course) { build :course, name: "Physics with mathematics" }
+
+        it { is_expected.to eq("Student finance if you’re eligible") }
```

Comment:
why is the test still passing?

Reply:
The assertion was correct, however this was because the subject factory did not have a bursary_amount. Have updated the factory and assertion to the following:

```
 context "Courses excluded from bursaries" do
        let(:mathematics) { build(:subject, :mathematics, bursary_amount: "3000") }
        let(:pe) { build(:subject) }

        let(:course) { build :course, name: "PE with mathematics", subjects: [mathematics, pe] }

        it { is_expected.to eq("Bursary or student finance if you’re eligible") }
end
``